tequilapi/client/dto.go

```diff
@@ -1,17 +1,24 @@
 package client
 
-// StatusDto holds connection status and session id
-type StatusDto struct {
+// StatusDTO holds connection status and session id
+type StatusDTO struct {
 	Status    string `json:"status"`
-	SessionId string `json:"sessionId"`
+	SessionID string `json:"sessionId"`
 }
 
-// IdentityDto holds identity address
-type IdentityDto struct {
+// StatisticsDTO holds statistics about connection
+type StatisticsDTO struct {
+	BytesSent       int `json:"bytesSent"`
+	BytesReceived   int `json:"bytesReceived"`
+	DurationSeconds int `json:"durationSeconds"`
```

Comment:
Should we add `..seconds` to attribute names?

Reply:
It specifies unit in which these values are returned. i.e.:
```
{
"sent": 23,
"received": 54,
"duration": 3
}
```
This would not be easy to understand, because it's unclear what units these values are in.